src/custom-element.js

```diff
@@ -1427,6 +1427,18 @@ function createBaseCustomElementClass(win, elementConnectedCallback) {
       return this.implClass_ ? this.implClass_.prerenderAllowed(this) : false;
     }
 
+    /**
+     * Whether the element can preview.
+     * @return {boolean}
+     * @final
+     */
+    previewAllowed() {
+      if (this.hasAttribute('nopreview')) {
```

Comment:
Will defer to wg-stories

Reply:
No, I don't think we would want something like that to be valid -- whether or not an element is previewable is defined by AMP's implementation, rather than the publishers'.

Reply:
Removed.